pkg/api/bzz.go

```diff
@@ -268,6 +272,14 @@ func (s *Service) fileUploadHandler(
 		ext.LogError(span, err, olog.String("action", "putter.Done"))
 		return
 	}
+	// TODO: what to do if act encrypt fails but the file is already stored ?
```

Comment:
Yes, the file is already stored at this point. The question is: do we want to upload the file unencrypted and give back that reference, in case act fails ?

Reply:
I don't understand why is it stored unencrypted.
If it was already uploaded before adding ACT to it, than we can't do anything about that.
When we are uploading something and we want it to be access controlled, I don't think there is a first step when it gets uploaded unencrypted.
It shouldn't be uploaded unencrypted.

Reply:
So do not even try to upload the unencrypted reference, only if ACT encryption succeeds ?

Reply:
I think yes. If possible, don't upload unencrypted data, because that might be totally the opposite what the user wants or expects. But I might misunderstand the original situation. For a new, access controlled upload, unencrypted reference shouldn't be uploaded, when failure.

Reply:
Okay, changed the encryption - upload order.